Source/DafnyLanguageServer.Test/Synchronization/IncludeFileTest.cs

```diff
@@ -25,17 +25,17 @@ public class IncludeFileTest : ClientBasedLanguageServerTest {
   }
 
   [Fact]
-  public async Task IndirectlyIncludedFileFails() {
+  public async Task IndirectlyIncludedFileFailsSemantic() {
     var source = @"
-include ""./includesSyntaxError.dfy""
-include ""./syntaxError.dfy""
+include ""./includesSemanticError.dfy""
+include ""./semanticError.dfy""
 ".TrimStart();
     var documentItem = CreateTestDocument(source, Path.Combine(Directory.GetCurrentDirectory(), "Synchronization/TestFiles/test.dfy"));
     await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
     var diagnostics = await diagnosticsReceiver.AwaitNextDiagnosticsAsync(CancellationToken);
     Assert.Single(diagnostics);
     Assert.Contains("the included file", diagnostics[0].Message);
-    Assert.Contains("syntaxError.dfy", diagnostics[0].Message);
+    Assert.Contains("semanticError.dfy", diagnostics[0].Message);
```

Comment:
I don't understand what this does. Does it check for the mention of `"semanticError.dfy"` in the error message? If that's all it does, then how does that test whether or not only one error message is being produced, as promised by the PR description?